source/read/distinct.txt

```diff
@@ -0,0 +1,160 @@
+.. _cpp-distinct:
+
+==============================
+Retrieve Distinct Field Values
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, unique, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to retrieve the
+distinct values of a specified field across a collection.
+
+Within a collection, different documents might contain different values for a
+single field. For example, one document in a ``restaurants`` collection has a
+``borough`` value of ``"Manhattan"``, and another has a ``borough`` value of
+``"Queens"``. Using the {+driver-short+}, you can retrieve all the unique values
```

Comment:
S: Because of the potential confusion of "using"

```suggestion
``"Queens"``. With the {+driver-short+}, you can retrieve all the unique values
```

Reply:
Changed to "by using" based on [this](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-using) style guide rec